docs/rst/getting_started/entities.rst

```diff
@@ -140,25 +140,25 @@ Physical Entities
 
 Host
 ----
-*Host* make reference to the physical (or logical i.e. Docker) machine where one or some DDS *Processes*
+A *Host* makes reference to the physical (or logical i.e. Docker) machine where one or more DDS *Processes*
```

Comment:
Add this `A` to every definition or do not add it here.

Reply:
:+1:

Reply:
NIT: Could you change Locator and DomainParticipant definitions so they start with *<entity>* as the others?

Reply:
:+1: